packages/runtime/src/useCases/transport/relationshipTemplates/LoadPeerRelationshipTemplate.ts

```diff
@@ -29,16 +30,16 @@ export class LoadPeerRelationshipTemplateUseCase extends UseCase<LoadPeerRelatio
     }
 
     protected async executeInternal(request: LoadPeerRelationshipTemplateRequest): Promise<Result<RelationshipTemplateDTO>> {
-        const result = await this.loadRelationshipTemplateFromReference(request.reference);
+        const result = await this.loadRelationshipTemplateFromReference(request.reference, request.password);
 
         await this.accountController.syncDatawallet();
 
         return result;
     }
 
-    private async loadRelationshipTemplateFromReference(reference: string): Promise<Result<RelationshipTemplateDTO>> {
+    private async loadRelationshipTemplateFromReference(reference: string, password?: string): Promise<Result<RelationshipTemplateDTO>> {
         if (reference.startsWith(Base64ForIdPrefix.RelationshipTemplate)) {
-            return await this.loadRelationshipTemplateFromRelationshipTemplateReference(reference);
+            return await this.loadRelationshipTemplateFromRelationshipTemplateReference(reference, password);
         }
 
         if (reference.startsWith(Base64ForIdPrefix.Token)) {
```

Comment:
you forgot to pass the password (next line)

I think you miss a test, this should have popped up in tests.